tools/vigiechiro/BilanEnrichiPFen.xml

```diff
@@ -0,0 +1,71 @@
+<tool id="vigiechiro_bilanenrichipf" name="Advanced restitution: 'Point fixe' protocol" version="@VERSION@">
+    <description>from Animal Detection on Acoustic Recordings</description>
+    <macros>
+        <import>vigiechiro_macros.xml</import>
+    </macros>   
+
+    <expand macro="vigiechiro_bilanPF_bilanRP_requirements" />
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript '$__tool_directory__/BilanEnrichiPF.R' 
+            '$idc2' 
+            '$__tool_directory__/refPF.csv' 
+            '$__tool_directory__/SpeciesList.csv' 
+        &&
+        tail -n +2 output-summary.html > tmp && mv tmp output-summary.html &&
+        tail -n +2 output-nightly.html > tmp && mv tmp output-nightly.html &&
+        tail -n +2 output-hourly.html > tmp && mv tmp output-hourly.html
+]]>
+    </command>
+    <inputs>
+        <param name="idc2" type="data" format="tabular" label="Corrected and validated observation file" help="Use output file from 'Tadarida identifications integration' : a sampling session file corrected and validated. Point fixe protocol"/>
+        <param name="output_choice" type="select" label="Select a summary display" help="Chose whether your want a nigthly, hourly or global summary">
```

Comment:
Can this be multiple="True"? So that I get 6 outputs?

Reply:
Maybe it will not be used that often, but yes I should enable multiple. Good point thank you